README.md

```diff
@@ -50,10 +51,6 @@ This package requires the [simpl-schema](https://github.com/aldeed/simple-schema
 - [Problems](#problems)
   - [SubObjects and Arrays of Objects](#subobjects-and-arrays-of-objects)
   - [Disable Displaying Collection Name in Error Message](#Disable-Displaying-Collection-Name-in-Error-Message)
-- [Contributing](#contributing)
```

Comment:
The removal seems unrelated, but I guess that's on purpose?

Reply:
Yes as these notes are already present in https://github.com/Meteor-Community-Packages/meteor-collection2/blob/master/CONTRIBUTING.md